lib/logger/lib/logger.ex

```diff
@@ -971,6 +965,8 @@ defmodule Logger do
     Enum.any?(matching, fn filter ->
       Enum.all?(filter, fn
         {:level_lower_than, min_level} ->
+          min_level = if min_level == :warn, do: :warning, else: min_level
```

Comment:
Please use `Logger.Handler.elixir_level_to_erlang_level(level)` instead. Once we deprecate it, we just need to deprecate it there.

Reply:
TBH unfortunately I cannot, as this will prevent using `level_lower_than: :emergency` for user that will decide to use new logger levels and I wanted to leave that option to the user, so I will only treat `:warn` as it was earlier.

It is not breaking change, as earlier there was no option to use levels above `:error` in Elixir, so it was not possible to define message that would omit `level_lower_than: :error`.

Reply:
`Logger.Handler.elixir_level_to_erlang_level(level)` simply converts :warn to :warning, everything else is returned as is. Or am I missing something?

Reply:
Ah, yeah. I have forgot. Changed.